src/digitKeyboard.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+import { EventDispatcher } from './eventDispatcher.js';
+
+/**
+ *  DigitKeyboard sets the digits on the screen.
+ */
+export class DigitKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindEvents();
+    }
+
+    /**
+     * Binds event listeners to digit buttons.
+     */
+    _bindEvents() {
+        let buttons = document.querySelector('.allButtons');
+        let numberButtons = buttons.querySelectorAll('.number');
+
+        for (let i = 0; i < numberButtons.length; i++) {
+            numberButtons[i].addEventListener("click", event => {
+                 let buttonDigit = Number(event.target.textContent);
+                 this.trigger('DigitKeyboard.KEY_PRESSED', buttonDigit)
```

Comment:
Perfect, you solved the root problem! Now the architecture is good.

The next thing which we need to improve are events names. It is a bad practice to use pure strings as event names, since it is prone to errors and hard to maintain. For example if you have:
```
bind('pera', doA());
bind('pera', doB());
bind('pera', doC());
bind('pera', doD());
// etc

trigger('pera');
```
And for some reason you need to change `pera` to `mika` you would have to update in all places, which is real world a big problem, since you can have hundreds of bindings for just one event name.

The rule of thumb is to store constants which some module can trigger in an object, which can then be used externally. For example in `calculator.js` you should be able to do:
```
digitKeyboard.bind(DigitKeyboard.KEY_PRESSED, digit => { screen.addDigit(digit); });
``` 
Notice that `DigitKeyboard.KEY_PRESSED` is field not a string in this example. With this approach, if you need to change `pera` to `mika`, you just update the object with constants and that is it.

Your task is to export constants from this module and use them in `calculator.js`.